conf/paths.go

```diff
@@ -18,17 +18,27 @@
 package conf
 
 import (
+	"os"
 	"path"
 )
 
 const (
 	BrokenArtifactSuffix = "_INCONSISTENT"
 )
 
+func getenv(key, fallback string) string {
+	value := os.Getenv(key)
+	if len(value) == 0 {
+		return fallback
+	}
+	return value
+}
+
 var (
-	// needed so that we can override it when testing
-	DefaultPathDataDir = "/usr/share/mender"
-	DefaultDataStore   = "/var/lib/mender"
+	// needed so that we can override it when testing and deployment on partially read-only systems
```

Comment:
Maybe we should add configuration variables for this to the configuration file as well? :thinking: 

What do you think @uishon ?

Reply:
It'd be nice indeed but that would require some more work/changes as some of these variables are used on startup, before the configuration is read. That's why I thought I'd go with environment variables for now.

Reply:
Alright, I think it's fine :smile_cat: